blockchain/beacon/src/drand.rs

```diff
@@ -54,9 +45,26 @@ where
     fn max_beacon_round_for_epoch(&self, fil_epoch: ChainEpoch) -> u64;
 }
 
+#[derive(SerdeDeserialize, SerdeSerialize, Debug, Clone)]
+pub struct ChainInfo {
+    public_key: String,
+    period: i32,
+    genesis_time: i32,
+    hash: String,
+    #[serde(rename = "groupHash")]
```

Comment:
is this really the case that they camel case this but not public_key and genesis_time?

Reply:
I'm not sure whats up with the docs, but heres the endpoint if you wanna see for yourself. It threw me off as well lol https://pl-eu.testnet.drand.sh/info

Reply:
Well, that wasn't thought through very well hahaha